src/payment/checkout/Checkout.jsx

```diff
@@ -168,6 +176,37 @@ class Checkout extends React.Component {
             {/* Apple Pay temporarily disabled per REV-927  - https://github.com/openedx/frontend-app-payment/pull/256 */}
           </p>
         </div>
+        {loadingCyberSource || loadingStripe
+          ? (
+            <>
+              <div className="skeleton py-1 mb-3 w-25" />
+              <div className="row">
+                <div className="col-lg-6">
+                  <div className="skeleton py-3 mb-3" />
+                  <div className="skeleton py-3 mb-3" />
+                  <div className="skeleton py-3 mb-3" />
+                  <div className="skeleton py-3 mb-3" />
+                </div>
+                <div className="col-lg-6">
+                  <div className="skeleton py-3 mb-3" />
+                  <div className="skeleton py-3 mb-3" />
+                  <div className="skeleton py-3 mb-3" />
+                  <div className="skeleton py-3 mb-3" />
+                </div>
+              </div>
+              <div className="skeleton py-1 mb-3 mt-5 w-25" />
+              <div className="row">
+                <div className="col-lg-6">
+                  <div className="skeleton py-3 mb-3" />
+                  <div className="skeleton py-3 mb-3" />
+                </div>
+                <div className="col-lg-6">
+                  <div className="skeleton py-3 mb-3" />
+                  <div className="skeleton py-3 mb-3" />
+                </div>
+              </div>
+            </>
+          ) : null}
```

Comment:
Could the below stripe and cybersource elements be moved to else of the ternary here instead of null since loading is already being checked as part of the conditional.

Reply:
I thought about that but wanted to avoid a nested ternary. But will try, maybe it will remove the issue that I have right now, when the loading skeleton is done but the Stripe fields don't load for a second (client secret is not null but the card fields take a while to load)

Reply:
Update: decided to continue not using a nested ternary (hard to read and also not recommended by the linter). I've instead created a method to return the loading skeleton to make it cleaner.